src/routes/sessions/[id]/+page.svelte

```diff
@@ -47,7 +45,6 @@
 	let messagesWindow: HTMLDivElement;
 	let userScrolledUp = false;
 
-	$: $editor.isNewSession = !$session?.messages.length;
```

Comment:
Not entirely sure what the issue was but I moved this line to [`handleSessionChange()`](https://github.com/fmaclen/hollama/pull/202/files#diff-e13c06b2f8eb5ca2f92007ee68d5fecf31e07131e5f14dcfb9a58c00efc01fe3R74) and it fixed #213.